modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java

```diff
@@ -393,6 +581,23 @@ protected List<DDMFormInstance> doFindByC_G_N_D_S(
 		}
 	}
 
+	protected String getPublishFilterQuery(boolean publish) {
+		StringBundler sb = new StringBundler(4);
+
+		sb.append(_PUBLISH_FILTER_QUERY_CONTEXT);
+		sb.append(_PUBLISH_FILTER_QUERY_RESTRICTIONS);
+		sb.append(Boolean.toString(publish));
+		sb.append("\"%'");
+
+		return sb.toString();
+	}
```

Comment:
@leticia-maciel For security reasons, could you try to move this to `custom-sql/default.xml` and try to replicate how it is done for the `name` and `description`, i.e., try to use the `_customSQL`, `queryPos` and whatever is necessary to not to create the query this way. Thanks!